rfcs/components/moving-from-focus-zone-to-tabster.md

```diff
@@ -0,0 +1,155 @@
+- `as?: React.ElementType`
+  - _Description:_ A component that should be used as the root element of the `FocusZone` component.
+  - _Why is no equivalent needed?_ There is no component to replace the root element of via `as`.
+- `shouldEnterInnerZone?: (ev: React.KeyboardEvent<HTMLElement>) => boolean`
+  - _Description:_ Callback function that will be executed on keypresses to determine if the user intends to navigate into the inner (nested) zone. Returning true will ask the first inner zone to set focus.
+  - _Why is no equivalent needed?_ Since `Tabster` is not a component but a set of utilities there is no concept of "nesting".
+- `allowFocusRoot?: boolean`
+  - _Description:_ Allows focus to park on root when focus is in the `FocusZone` at render time.
+  - _Why is no equivalent needed?_ There is no component with a root to park at.
+- `shouldFocusInnerElementWhenReceivedFocus?: boolean`
+- `shouldResetActiveElementWhenTabFromZone?: boolean`
+- `checkForNoWrap?: boolean`
+
+#### Props that may need an equivalent
+
+##### IFocusZone
+
+- `focusElement(childElement?: HTMLElement, forceAlignment?: boolean): boolean`
+  - _Description:_ Sets focus to a specific child element (`childElement`) within the zone. This can be used in conjunction with `shouldReceiveFocus` to create delayed focus scenarios (like animate the scroll position to the correct location and then focus). If `forceAlignment` is true, then focus alignment will be set according to the element provided. Returns true if focus could be set to an active element and false otherwise.
+  - _What should we do about this prop?_ There is probably not a need to have something that matches this functionality 1:1. However, forcing alignment and the ability to specify a child element to get focus are probably things that we want to have something to transition to.
+- `setFocusAlignment(point: Point): void`
+  - _Description:_ Forces horizontal alignment in the context of vertical arrowing to use a specific point (`point`) as the reference, rather than a center based on the last horizontal motion.
+  - _What should we do about this prop?_ We may need something similar to force horizontal alignment, although I am not so sure about using a specific point as reference.
+
+##### IFocusZoneProps
+
+- `defaultTabbableElement?: string | ((root: HTMLElement) => HTMLElement)`
+  - _Description:_ Optionally defines the initial tabbable element inside the `FocusZone`. If a string is passed then it is treated as a selector for identifying the inital tabbable element. If a function is passed then it uses the root element as a parameter to return the initial tabbable element.
+  - _What should we do about this prop?_ Is there an actual need for something like this? If so, we should probably find a solution in `Tabster`. If not, we should skip this prop and regard it as "not needed".
+- `onActiveElementChanged?: (element?: HTMLElement, ev?: React.FocusEvent<HTMLElement>) => void`
```

Comment:
is this equivalent to the native `onFocus` event handler ?

Reply:
It serves a similar purpose but also giving you the exact element that is active as part of the callback.